app/mixins/keyboard-shortcuts-global.js

```diff
@@ -48,8 +48,11 @@ export default Mixin.create(EKMixin, {
 	toggleShortcutsModal: on(keyUp('shift+Slash'), function() {
 		this.toggleProperty('uiStates.showShortcutsModal')
 	}),
-	focusSearchInput: on(keyUp('Slash'), function() {
-		document.querySelector('.InputAutocomplete .aa-input').focus()
+
+	focusSearchInput: on(keyDown('Slash'), function(event) {
```

Comment:
omg! ty

Reply:
why though keyDown, sometimes I am remotely incapable of understanding why

Reply:
it's the web 🤷‍♂️

Reply:
![50014544_306923893291641_7822178740816838656_n](https://user-images.githubusercontent.com/1502484/51445063-e292fc80-1d00-11e9-94b9-c905dd0cf63e.jpg)